code/modules/unit_tests/open_air.dm

```diff
@@ -0,0 +1,29 @@
+/datum/unit_test/open_air/Run()
+	var/datum/virtual_level/vlevel = mapzone.virtual_levels[1]
+	for(var/turf/open/T in vlevel.get_block())
+		T.Initalize_Atmos(0)
+		T.air.clear()
+	var/turf/open/center_turf = vlevel.get_center()
+	center_turf.air.set_moles(GAS_PLASMA, 32)
+	for(var/i in 1 to 10)
+		SSair.fire()
+		sleep()
```

Comment:
I don't believe calling sleep will resolve issues where ssAir will early return itself, I might be incorrect in that assumption however

Reply:
I might just have a wrong idea of how the internal scheduler works, yeah--sleep(1) it is then